src/main/java/org/opentestsystem/ap/imrt/common/model/BaseEntity.java

```diff
@@ -49,4 +49,18 @@ public String getUpdatedBy() {
     public void setUpdatedBy(String updatedBy) {
         this.updatedBy = updatedBy;
     }
+
+    /**
+     * @param createdAt to set the created at.  During persistence this will get set by the ORM
+     */
+    public void setCreatedAt(final Instant createdAt) {
```

Comment:
I added setters for these so we can use them during tests.  Prior to this addition the only way to set them is via reflection or the ORM.